docs/plugins.rst

```diff
@@ -307,6 +307,10 @@ any of the available LDAP servers, they will fall back to normal core
 authentication. Documentation of the LDAP standard in general can be found
 `here <https://en.wikipedia.org/wiki/Lightweight_Directory_Access_Protocol>`_.
 
+Installation of this plugin requires some system libraries:
```

Comment:
"Installation of this plugin requires LDAP and SASL shared libraries to be installed and available to the Girder process. These may be installed system-wide via package managers in the following way:"

Reply:
@zachmullen Can you just make an inline "change suggestion", with the GH? I'll be happy to apply it.